packages/platform/examples/base/src/pages/index.tsx

```diff
@@ -28,6 +28,7 @@ export default function Home(props: { calUsername: string; calEmail: string }) {
               redir="http://localhost:4321/calendars"
               className="h-[40px] bg-gradient-to-r from-[#8A2387] via-[#E94057] to-[#F27121] text-center text-base font-semibold text-transparent text-white hover:bg-orange-700"
             />
+            <Connect.AppleCalendar className="h-[40px] bg-gradient-to-r from-[#8A2387] via-[#E94057] to-[#F27121] text-center text-base font-semibold text-transparent text-white hover:bg-orange-700" />
```

Comment:
After connecting apple calendar I did get confirmation that it was connected (and indeed i made a booking and event showed up in my apple calendar) but the connect Apple Calendar button still allows me to enter credentials again even after hard refreshing the page:

<img width="1709" alt="Screenshot 2024-06-26 at 10 15 32" src="https://github.com/calcom/cal.com/assets/42170848/e5a48300-b29a-4c23-8954-19e2ff212b75">

Reply:
@supalarry indeed there was an issue like this, and it was happening for all calendars including outlook as well as google calendars. The issue was the enabled property in the useCheck hook was not set correctly, but thanks to help from @ThyMinimalDev it's resolved now in commit [fixup](https://github.com/calcom/cal.com/pull/15510/commits/09141b40c678a11cba17ae4fa40d933129aa0c10). Can you try again now?

Reply:
Also, if invalid credentials are entered we return undefined error but could inform user that credentials are wrong:
<img width="1689" alt="Screenshot 2024-06-26 at 14 46 17" src="https://github.com/calcom/cal.com/assets/42170848/c3d4842c-af13-44ef-a37e-bb6b967b267e">